test/pom.xml

```diff
@@ -52,6 +52,9 @@ THE SOFTWARE.
 
     <!-- Filled in by maven-hpi-plugin with "-javaagent:/path/to/mockito-core-<version>.jar" -->
     <jenkins.javaAgent />
+
+    <!-- Faster incremental build https://github.com/jenkinsci/jenkins/pull/11071#issuecomment-3368149474 -->
+    <test>hudson.console.ConsoleAnnotatorTest</test>
```

Comment:
:warning:

Reply:
Reverted in aa53098855861b059225c2199fa11d7852f6a41e.